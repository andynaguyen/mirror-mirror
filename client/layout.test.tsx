import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Layout from './Layout';

describe('<Layout />', () => {
  it('should render the contents of the layout component', () => {
    const component = renderer.create(<Layout />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
