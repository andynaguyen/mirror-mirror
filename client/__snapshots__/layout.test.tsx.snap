// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should render the contents of the layout component 1`] = `
<div
  className="sc-bdVaJa gWRKSQ"
>
  Main Layout
</div>
`;
